src/Future.ts

```diff
@@ -19,6 +21,60 @@ declare module 'fp-ts/lib/HKT' {
  */
 export const URI = 'Fluture/Future'
 
+/**
+ * @since 0.6.4
+ */
+export interface Future<E, A> extends F.FutureInstance<E, A> {}
+
+/**
+ * @since 0.6.4
+ */
+export const left: <E = never, A = never>(e: E) => Future<E, A> = F.reject
+
+/**
+ * @since 0.6.4
+ */
+export const right: <E = never, A = never>(a: A) => Future<E, A> = F.resolve
+
+/**
+ * @since 0.6.4
+ */
+export function orElse<E, A, M>(onLeft: (e: E) => Future<M, A>): (ma: Future<E, A>) => Future<M, A> {
+  return ma => F.chainRej(onLeft)(ma)
+}
+
+/**
+ * @since 0.6.4
+ */
+export function fork<E, A>(onLeft: (e: E) => void, onRight: (a: A) => void): (ma: Future<E, A>) => () => void {
+  return ma => F.fork(onLeft)(onRight)(ma)
+}
+
+/**
+ * @since 0.6.4
+ */
+export const value = F.value
+
+/**
+ * @since 0.6.4
+ */
+export const coalesce = F.coalesce
```

Comment:
This is similar to [fold](https://gcanti.github.io/fp-ts/modules/TaskEither.ts.html#fold).
If the goal is to improve interoperability, maybe you could use `Task` as return types